app/src/main/res/values/styles.xml

```diff
@@ -11,6 +11,70 @@
         <item name="actionModeCloseDrawable">@drawable/ic_close</item>
     </style>
 
+    <style name="AppTheme_pink" parent="Theme.AppCompat.DayNight.NoActionBar">
```

Comment:
All the themes have a few small glitches with some of the colours.

Background colour is wrong:

| Catima | Pink |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1885159/130276563-3b1d21f4-3ab6-46b4-9bcc-853ebbcf6605.png) | ![image](https://user-images.githubusercontent.com/1885159/130276583-de75fca1-f16e-4c75-b8bd-6488affa4d63.png) |

Dialog buttons are pure black (also, this is the pink theme, why is the text field blue?):
| Catima | Pink |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1885159/130276822-a4c84377-5bcc-4e02-941f-fb53b6fb6ac3.png) | ![image](https://user-images.githubusercontent.com/1885159/130276867-2f6abf87-d3b1-4061-9645-a33066b20726.png) |

Reply:
what should be the background color?
text field blue due to color_accent ok I will do it pink.
also I make dialog buttons in theme color, is it good?

Reply:
I think if you set the parent to AppTheme for each of them it should probably fix the background colors.

Using the pink for buttons on pink theme sounds good yes

Reply:
I am not getting these different background colors in my phone. In my phone background color is white how you got these background color colors?

Reply:
apart from that, I fixed dialog buttons.

Reply:
I tested it at night, so I was using the Dark theme. If you set your theme (not theme colors) to dark you should see it too.

Reply:
ok, I did all the changes.

Reply:
The button colours are still broken.

I also noticed multi-select is buggy on all the non-Catima themes:
| Catima | Pink |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1885159/130320445-98e16a5e-db3b-424a-89da-24e49cc2e48c.png) | ![image](https://user-images.githubusercontent.com/1885159/130320446-13a39807-ec98-4ea7-843f-62a77a634b9d.png) |

Can you please set every theme parent to AppTheme? So not:

```
    <style name="AppTheme_pink" parent="Theme.AppCompat.DayNight.NoActionBar">
```

but

```
    <style name="AppTheme_pink" parent="AppTheme">
```

I think that should solve a lot of issues too. Then we don't need `android:windowBackground` in every single theme.

Reply:
ok I will do it.